packages/plugin-collapsible-list/web/src/tiptap/component.tsx

```diff
@@ -10,6 +11,24 @@ import { TIPTAP_COLLAPSIBLE_ITEM_TYPE } from 'ricos-content';
 import { findParentNodeClosestToPos, isInCollapsibleList } from './utils';
 import { RicosContext } from 'ricos-context';
 import classNames from 'classnames';
+import { collapsibleStateManagerPlugin } from '../consts';
+
+const findContainerNode = (editor, nodeId) => {
+  const parentNode = editor.state.doc.content.content
+    .filter(currNode => currNode.type.name === 'COLLAPSIBLE_LIST')
+    .find(currNode => currNode.content.content.some(item => item.attrs.id === nodeId));
+  return parentNode;
+};
+
+const isInContainer = (parentNode, itemId) => {
+  return parentNode.content.content.some(node => node.attrs.id === itemId);
+};
+
+const TO_RICOS_OPTIONS = {
+  first_expanded: 'FIRST',
+  expanded: 'ALL',
+  collapsed: 'NONE',
```

Comment:
```suggestion
  [FIRST_EXPANDED]: 'FIRST',
  [EXPANDED]: 'ALL',
  [COLLAPSED]: 'NONE',
```

import { EXPANDED, COLLAPSED, FIRST_EXPANDED } from '../defaults'

Reply:
done